apps/assets/Access-Legacy-Code.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="docs.hpi-schul-cloud.org" modified="2020-09-17T08:39:16.937Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36" etag="Ftq0aKvM3ce6FppLR7yL" version="13.6.6" type="atlas"><mxAtlasLibraries/><diagram id="RrnGyhTox1kCKPTknkRC" name="Seite-1">7VvbdqM2FP0ar9U+mIUkEPAY32amddtM02mbvmFQMB1suSAn9nx9JZBsZOFLxzhxM8FrxdZBEnD23keXQzqoP1u9y8PF9Ccak6wD7XjVQYMOhChwbf4lLOvKAqAbVJYkT2Np2xru0i9EGmXDZJnGpNAqMkozli50Y0TncxIxzRbmOX3Sqz3QTL/qIkzkFe2t4S4KM2JU+yON2bSy+tDb2t+TNJmqKwMsn28Wqsqy42IaxvSpZkLDDurnlLLq12zVJ5nwnvJL1W605+zmxnIyZ6c04O4Fq3zQDX6MH778/tfjD/ef3K6Dqm4ew2wpn1jeLVsrFyQ5XS46qPdA50wCxNFBPfMG5D09kpyRVRM84SQjux7g3CF0Rli+5vVkqy5EWLpNEqermPS0RcF1pG1aQ8BByELyqUIJfrK5wNY//Id00X9wV2B46/3tB264i6bLrNvP6DLuiEfDGb+b3iTXHIn/WQqsSz92i9KRN7wCsBer0pvqPP+VyO+qm7N6uSM5x0P1xZ96su1/B2iO8zwm4vkFvE/TlJG7RRiJs09c3tw2ZbNMwN/AhjBLkzkvZORBdC44kHIZ3Ugzo6J9wXL6mfRpRnNum9M5ET2lWaZMHYhGUHwa+XWYwrusM9klueRCg0qO3UAl5IPLUUkFuBqXxjRJo/b4Mw7XAvhzQN4BteAV03kyLksDfCKgvf7QG41OCiDnAizPQsfCSAsf0EYW9gzYQeBayDOR9/xLoQ4M1IVAU+7/1wX6cDgIRsHLgg7ANSAODcQHIQtbgvsmikhRvDLujEYDPOi9cMAIgGUOE/vYgx0LXGygMKdonwqBk92nswV3IPdFK5hFvCfeL+pRgRoTjoC2iQ5CNj+4PQsnJLulRcpSqrVXY/94p8IsjWNxh3smByXa8hmaZhiN1LkYJbBrYd/h6xAXYgRcX2OHD/nEADZEF+BYgeOhbUuTKzCweWM1ELRPF8dgA4n5kkYWac6mNKHzMBturT2dL9s6YyqQKdH4mzC2lnCES0Z1DpFVyv4UzS1Xlu5rZwYr2XNZWMvCmeoWT3UQyJxkIUsf9YVck79l01uaCikpAgAvsHwFv+dzWPUA4fkWCDwf6/0WdJlHRHZVX4210jsL84Qwo/eSKJsnP4M7eF+o2UxSWg00RmQRh2SGNh6I4znHAw9ZvqZjfV6BhYIVo2ryD3yrBihv3aB+1wqC4GKTDe9FxD/n915TvyjeK8WLwlb/ZamlAFBJ7ZA3pJsr1ZzA+/ZCynko+vtk+CspRAy5EZsOb0ostxMtjF1zID5BicDjIkYg2ByXEiU0V/oSTvHVDwvyXfH9G57VPBzz8OhfYWRVYn/eyHrpCKl2e49GSMXhK4mQ6r51SQkpceMojML4bapSiQKIlQrAu0HuSkMlcF+jytT04rjK4HWpzNk/D1mIZT0tb6YDcTgT+qn+ynGNfw3njC/9SfEmRXEWAduC2IFfpcALLxuC1yg7NWYdl51zVbJDVzHV2KemtlFCp05B1KroWlAypyBVLrifpdw133rQu6bwhlwDKpGCmZTTxR2Uimm4ED+jdZZyuHJ4HKtJBex4sjGE0eekhPuXJePdqA3tKgVTpkF0MhwC9xRNGmzRt8pFOq4nPg1b75s9eZmq+a0MF/iSjPGhpW18Ovow6QecC+aKfk8rLV/vWBhjtJ2m4gsRCiKDN684Qqvh8WiEVjp7/gg9HH94/PjzbH07vyXMTd8NHf9D10y1j0kSRnLKukmYVrlP+epav77XLQrvKf38zcRy6NuWbddkpimzOSV6QnRXr3K1jrApwzYTWTiopbJsK/DRwXSWEXN9W3wa8BPNbkmecicIPlRdHY0ftV32IND32S18ZKe9LO1e8ow4cUhvLxAmmlNsEAeWV5uZ+DsDjYPFesz21Gih939yIg8Dqz4uIT2R54Ad8reXwmsEwcz+tpje0UnnHiHdqeOVkhzQ5OYeFFvrY1zDWvEQyS/NXWQHllt7jQBopAKeeMcV2uoAX0ldHxzcO0Qi0vhwM53ynpXJ5lbg1SQq22Ty/mGjbY43bUNeL8ehK9YB/objtvv/Ijkvbt/tr6pv/0UCDf8F</diagram></mxfile>
```

Comment:
The assets are not part of the final build or? It is hard to find the build file by this many changes.

Reply:
thay are, that are just pictures/drawio files however here, we not need to check the code but the final build svg file only. this is to easily update the svg based on drawio files like that.